client/state/billing-transactions/reducer.js

```diff
@@ -46,6 +49,48 @@ export const requesting = createReducer( false, {
 	[ BILLING_TRANSACTIONS_REQUEST_SUCCESS ]: () => false,
 } );
 
+/**
+ * Returns an updated subtree for individual receipts that are not found in the receipt list
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export const individualTransactions = createReducer(
+	{},
+	{
+		[ BILLING_RECEIPT_REQUEST ]: ( state, { receiptId } ) => ( {
+			...state,
+			requesting: {
+				...state.requesting,
+				[ receiptId ]: true,
+			},
+		} ),
+		[ BILLING_RECEIPT_REQUEST_FAILURE ]: ( state, { receiptId, error } ) => ( {
+			...state,
+			errors: {
```

Comment:
Seems like using the notices middleware could prevent us from storing the errors in the Redux state completely.